site/content/troubleshooting-guide.md

```diff
@@ -1 +1,122 @@
-### TODO
\ No newline at end of file
+
+# Troubleshooting Guide
+
+This guide outlines some failure scenarios and their possible solutions.
+
+## Node.js not installed
+AWS.Deploy.Tools relies on AWS CDK to provision resources for the user's cloud application. AWS CDK requires Node.js to be installed in the user's machine.
+
+**Minimum Node.js version >= 10.13.0**
+
+See [here](https://nodejs.org/en/download/) to install Node.js on your system.
+
+## Docker related issues
```

Comment:
Since we generate the dockerfile and include all of the dependencies of the project that we detect at the time. They will need to update the Dockerfile when they add new class library project dependencies. It would be a good idea to talk about how we generate the Dockerfile so the understand what to update.

Reply:
Added